src/views/TabView/TabBarBottom.js

```diff
@@ -39,6 +39,7 @@ type Props = {
   getLabelText: (scene: TabScene) => string;
   renderIcon: (scene: TabScene) => React.Element<*>;
   style: any;
+  labelStyle: any?;
```

Comment:
Does this actually pass flow? Usually the `?` comes before the type, but I don't think its needed for `any`.